composer.json

```diff
@@ -92,7 +98,7 @@
         "doctrine/doctrine-fixtures-bundle": "^3.0.2",
         "doctrine/doctrine-migrations-bundle": "^1.3.0",
         "doctrine/migrations": "^1.6.0",
-        "egeloen/ordered-form": "^3.0",
+        "egeloen/ordered-form": "^4.0",
```

Comment:
this should be updated in `packages/framework/composer.json` too

Reply:
Fixed